graphql-protocol/src/main/java/com/linecorp/armeria/server/graphql/protocol/AbstractGraphqlService.java

```diff
@@ -152,24 +152,25 @@ protected abstract HttpResponse executeGraphql(ServiceRequestContext ctx, Graphq
         if (Strings.isNullOrEmpty(value)) {
             return ImmutableMap.of();
         }
-        return parseJsonString(value);
+        return toMapFromJson(parseJsonString(value));
```

Comment:
Should we need to additionally call `toMapFromJson()`?

Reply:
no, that's just wrong. Fixing!